velox/type/Variant.cpp

```diff
@@ -187,7 +187,14 @@ std::string variant::toJson(const TypePtr& type) const {
     return '"' + DATE()->toString(value<TypeKind::INTEGER>()) + '"';
   }
 
-  switch (kind_) {
+  VELOX_USER_CHECK_EQ(
+      this->kind(),
+      type->kind(),
+      "variant TypeKind {} does not match input TypeKind {}",
```

Comment:
No need to include type kinds in the message. These are included by default.

Reply:
Fixed.